Tools/Scripts/jsc-stress-test-helpers/shell-runner.sh

```diff
@@ -67,6 +68,12 @@ do
         lock_test_list
         while [ -s ${testList} ]
         do
+            if [ -e ${pauseFile} ]
+            then
+                echo 'Running paused...'
```

Comment:
Printing this every 5 seconds seems very spammy?

Reply:
change to 10 minutes, since those will run in the background anyway.